lib/miq_automation_engine/service_models/miq_ae_service_physical_server.rb

```diff
@@ -4,5 +4,12 @@ class MiqAeServicePhysicalServer < MiqAeServiceModelBase
     expose :turn_off_loc_led
     expose :power_on
     expose :power_off
+
+    def self.create_server_profile_and_deploy_task(ems_id, template_id, server_id, profile_name)
+      ext_management_system = ExtManagementSystem.find(ems_id)
```

Comment:
Hm according to https://github.com/ManageIQ/manageiq/pull/22241/files#diff-8d286308b75ae7dc8a266753732b0c1709d33293518c479765e34cadf8207910R90 a profile_template belongs to an ext_management_system so this shouldn't be required.  If you need access to the EMS you can reference it from the `profile_template` instance